service-worker.js

```diff
@@ -0,0 +1,19 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+var swOptions = {
+  debug: true
+}
+
```

Comment:
TL for sw-toolbox and sw-precache here. Great to see you using it :)

If you're going to set `maxEntries` to 20, it means that after the 21st entry is cached, the least-recently used entry would be automatically deleted. Unless you're storing a number of really large assets, I'd consider whether this is really that valuable. I don't think the Babel site would benefit from setting this all that much.

I do think there's use in imposing a maxAgeSeconds age. @boopathi can probably comment to the length of time, but a week doesn't sound like a bad ballpark.